src/main/scala/it/pagopa/interop/catalogprocess/service/impl/CatalogManagementServiceImpl.scala

```diff
@@ -25,7 +25,9 @@ final case class CatalogManagementServiceImpl(invoker: CatalogManagementInvoker,
   implicit val logger: LoggerTakingImplicit[ContextFieldsToLog] =
     Logger.takingImplicit[ContextFieldsToLog](this.getClass)
 
-  override def createEService(eServiceSeed: EServiceSeed)(implicit contexts: Seq[(String, String)]): Future[EService] =
+  override def createEService(
+    eServiceSeed: EServiceSeed
+  )(implicit contexts: Seq[(String, String)]): Future[EService] =
```

Comment:
As a general rule of thumb, don't reformat code if it's not strictly needed

Reply:
+1
if you reformat this code the change should disappear